GameOfLifeApp/Controllers/ApplicationManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Manages a game.
+    /// </summary>
+    public class ApplicationManager : IApplicationManager
+    {
+        private IGameManager manager;
+
+        public ApplicationManager(IGameManager gameManager)
+        {
+            manager = gameManager;
+        }
+
+        /// <summary>
+        /// Manages the running app.
+        /// </summary>
+        public void RunApplication(IGameField field, ICommunicator chat, ISerializer converter)
+        {
+            field = new GameField();
```

Comment:
why do you pass the field here, and create a new one?
If you always need to create a new field, then there is no reason to pass it into the method.